### PR TITLE
Initial JupyterLab package

### PR DESCRIPTION
**Pull request by jasongrout**
Here's an initial JupyterLab package, mirroring the pypi one.

We expect to update this rather frequently in the near future.

### CODE REVIEW

**Comment by jasongrout** [comment]
@blink1073 - I added you as a maintainer. Are you okay with that?

File: recipes/jupyterlab/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+about:
+  home: https://github.com/jupyter/jupyterlab
+  license: BSD 3-Clause
+  summary: A pre-alpha version of a computational environment for Jupyter.
+
+extra:
+  recipe-maintainers:
+    - jasongrout
+    - blink1073
```
Comment (patricksnape) [inline recipes/jupyterlab/meta.yaml]: Missing new line, otherwise recipe looks really good.

**Comment by blink1073** [comment]
@jasongrout, yep, happy to be listed.

File: recipes/jupyterlab/meta.yaml (hunk 1 above)
Comment (jasongrout) [inline recipes/jupyterlab/meta.yaml]: Fixed.

**Comment by ocefpaf** [comment]
@jasongrout based on the linter message I recon you are working in an outdated fork.

> I just wanted to let you know that I linted all conda-recipes in your PR (```recipes/decorator```, ```recipes/jupyterlab```) and found it was in an excellent condition.

Please rebase this PR against latest master so we can get rid of `decorator`.

File: recipes/jupyterlab/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    url: https://pypi.python.org/packages/a9/92/bd00eb71efbdc47b6a3b555a60a0f6ba8c8cb9b27aa6927b9242ad35c5fc/jupyterlab-{{ version }}.tar.gz
+    md5: ba625d966b7d11340adac7776ba123cc
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
```
Comment (jakirkham) [inline recipes/jupyterlab/meta.yaml]: Please add `setuptools`.

**Comment by jasongrout** [comment]
Rebased.

File: recipes/jupyterlab/meta.yaml (hunk 2 above)
Comment (jasongrout) [inline recipes/jupyterlab/meta.yaml]: Fixed. Thanks!

**Comment by ocefpaf** [comment]
Thanks @jasongrout! :tada:

**Comment by jasongrout** [comment]
Yeah! Thanks!

**Comment by ocefpaf** [comment]
Sorry it took so long. Sometimes it is worth pinging us again to re-raise PR in our mailboxes. Things get buried easily here.

**Pull request merged**